cluster/gce/util.sh

```diff
@@ -1302,6 +1302,7 @@ OPENCONTRAIL_TAG: $(yaml-quote ${OPENCONTRAIL_TAG:-})
 OPENCONTRAIL_KUBERNETES_TAG: $(yaml-quote ${OPENCONTRAIL_KUBERNETES_TAG:-})
 OPENCONTRAIL_PUBLIC_SUBNET: $(yaml-quote ${OPENCONTRAIL_PUBLIC_SUBNET:-})
 E2E_STORAGE_TEST_ENVIRONMENT: $(yaml-quote ${E2E_STORAGE_TEST_ENVIRONMENT:-})
+KUBELET_PORT: $(yaml-quote ${KUBELET_PORT:-10250})
```

Comment:
Please leave the default empty instead of 10250. If that breaks cluster creation, it should be fixed elsewhere rather than putting a magic number here.